Documentation/check-codeowners.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+source_dir="$(cd "${script_dir}/.." && pwd)"
+target_file="${script_dir}/codeowners.rst"
+
+if ! git diff --quiet -- "$target_file" ; then
+    git --no-pager diff "$target_file"
```

Comment:
```suggestion
    git --no-pager diff -- "$target_file"
```
for consistency with the line above